neo/IO/Helper.cs

```diff
@@ -129,12 +129,12 @@ public static T[] ReadSerializableArray<T>(this BinaryReader reader, int max = 0
             return array;
         }
 
-        public static byte[] ReadVarBytes(this BinaryReader reader, int max = 0X7fffffc7)
+        public static byte[] ReadVarBytes(this BinaryReader reader, int max = 0x1000000)
         {
             return reader.ReadBytes((int)reader.ReadVarInt((ulong)max));
         }
 
-        public static ulong ReadVarInt(this BinaryReader reader, ulong max = ulong.MaxValue)
+        public static ulong ReadVarInt(this BinaryReader reader, ulong max = 0x1000000)
```

Comment:
`ReadVarInt` is not only used for `ReadVarBytes`, but also for reading ordinary integers.

Reply:
You are right :) thanks!